wrangler-transform/src/main/java/io/cdap/wrangler/Wrangler.java

```diff
@@ -189,11 +190,11 @@ public void configurePipeline(PipelineConfigurer configurer) {
       }
 
       if (!config.containsMacro(Config.NAME_PRECONDITION_LANGUAGE)) {
-        if (PRECONDITION_LANGUAGE_SQL.equalsIgnoreCase(config.getPreconditionLanguage())) {
+        if (checkSQLExecution(config)) {
```

Comment:
These changes will allow users to run directives in SQL along with the existing precondition feature flag.